README.md

```
@@ -4,7 +4,7 @@
   <br>
 </h1>
 
-<h3 align="center">New Repository Template</h3>
+<h3 align="center">Binancial</h3>
```

Markdownlint (0.37.0)

Element: h3
Inline HTML

(MD033, no-inline-html)